apps/testnorge-arena/arena-core/src/test/java/no/nav/registre/arena/core/provider/rs/AapControllerIntegrationTest.java

```diff
@@ -1,31 +1,273 @@
+    private static final String miljoe = "test";
+    private static final Long avspillergruppeId=123456789L;
+
+    private static final String tokenProviderUrl="(.*)/token-provider";
+    private static final String hentLevendeIdenterIAldersgruppeUrl="(.*)/hodejegeren/api/v1/levende-identer-i-aldersgruppe/"+avspillergruppeId;
+    private static final String hentIdenterIAktoerregisteretUrl="(.*)/aktoerregister/v1/identer";
+    private static final String brukereArenaUrl="(.*)/arena-forvalteren/api/v1/bruker";
+    private static final String hentSyntetisertRettighetAapUrl="(.*)/syntetisereren/api/v1/arena/aap";
+    private static final String hentSyntetisertRettighetAap115Url="(.*)/syntetisereren/api/v1/arena/aap/11_5";
+    private static final String opprettPersonIPoppUrl="(.*)/pensjon-testdata-facade/api/v1/person";
+    private static final String opprettInntektIPoppUrl="(.*)/pensjon-testdata-facade/api/v1/inntekt";
+    private static final String opprettRettigheterAap115Url = "(.*)/arena-forvalteren/api/v1/aap115";
+    private static final String opprettRettigheterAapUrl = "(.*)/arena-forvalteren/api/v1/aap";
+    private static final String saveHistorikkUrl="(.*)/hodejegeren/api/v1/historikk";
+
+    private NyttVedtakAap vedtak;
+    private NyttVedtakResponse nyttVedtakResponse;
+    private NyeBrukereResponse nyeBrukereResponse;
+    private Map<String, AktoerResponse> identerIAktoerRegisteret;
+    private PensjonTestdataResponse pensjonTestdataResponse;
+
+    @Before
+    public void setUp(){
+        vedtak = NyttVedtakAap.builder().aktivitetsfase("TEST").build();
+        vedtak.setFraDato(LocalDate.now().minusMonths(3));
+        vedtak.setTilDato(LocalDate.now());
+
+        nyttVedtakResponse = NyttVedtakResponse.builder()
+                .nyeRettigheterAap(Collections.singletonList(vedtak))
+                .feiledeRettigheter(Collections.emptyList())
+                .build();
+
+        nyeBrukereResponse = new NyeBrukereResponse();
+        nyeBrukereResponse.setAntallSider(0);
+        nyeBrukereResponse.setArbeidsoekerList(Collections.emptyList());
+
+        identerIAktoerRegisteret = new HashMap<>();
+        var aktoerResponse = new AktoerResponse();
+        aktoerResponse.setIdenter(Collections.singletonList(new AktoerInnhold(ident,"test", true)));
+        identerIAktoerRegisteret.put(ident, aktoerResponse );
+
+        pensjonTestdataResponse = PensjonTestdataResponse.builder()
+                .status(Collections.singletonList(PensjonTestdataStatus.builder()
+                        .miljo(miljoe)
+                        .response(PensjonTestdataResponseDetails.builder()
+                                .httpStatus(new HttpStatus("test", 200))
+                                .build())
+                        .build()))
+                .build();
+
+    }
 
-    @Ignore
     @Test
-    void test() {
-        // Todo: implementer
+    public void shouldGenerereRettighetAap() throws Exception {
```

Comment:
Tenker denne appenen er for kompleks når testene blir så store :)

Reply:
Helt enig, burde kanskje se på om det er mulig å forenkle applikasjonen